mtp_bank_admin/apps/bank_admin/refund.py

```diff
@@ -35,5 +36,8 @@ def generate_refund_file(request):
     client = api_client.get_connection(request)
     client.bank_admin.transactions.patch(refunded_transactions)
 
+    # create file record
+    post_new_file(request, 'ACCESSPAY', [t['id'] for t in refunded_transactions])
```

Comment:
Yeah, I guess. I was trying to avoid thinking of these terms as constants. I don't have a real reason for that, though.